admin_guide/install/deploy_router.adoc

```diff
@@ -95,7 +95,9 @@ $ oadm router <router_name> --replicas=<number> \
 ----
 endif::[]
 
-For example:
+For example, using the credentials the OpenShift master generates
+by default for the router at startup and creating two instances on
+different hosts:
```

Comment:
"For example, to create two instances on different hosts using the router credentials the master generates
by default at startup:"